@types/route.d.ts

```diff
@@ -8,5 +8,5 @@ type AppRoute =
   | '/auth/login'
   | '/dashboard'
   | '/dashboard/families'
-  | '/dashboard/families/family-detail'
-  | '/dashboard/projects';
+  | '/dashboard/projects'
+  | 'dashboard/families/:familyId';
```

Comment:
it's better to use kebab-case in url path since it's case-insensetive
```suggestion
  | 'dashboard/families/:family-id';
```

Reply:
can't we just do: 
```
Params: {
    family-id: string;
  };
```

Reply:
Nope. It's not too important.

Reply:
Let's go with `/:id`